src/config/mod.rs

```diff
@@ -1314,6 +1314,15 @@ impl DbConfig {
                 self.writecf.disable_write_stall = true;
                 self.lockcf.disable_write_stall = true;
                 self.raftcf.disable_write_stall = true;
+                // Effectively disable flush by size.
+                self.defaultcf.write_buffer_size = ReadableSize::gb(1024);
+                self.writecf.write_buffer_size = ReadableSize::gb(1024);
+                self.lockcf.write_buffer_size = ReadableSize::gb(1024);
```

Comment:
I understand this change is to trigger the rocksdb's flush logic based on age and size. However, it has some negative impacts. For example, if there's a hot region write, then the memtable size effectively is the max sum of write buffer (10GB), which is too big for fast recovery.
Also the way it works is a hack to me (it solely depends on the logic of how rocksdb's flush works)

Why not flush old memtable in TiKV side? It's more flexible and does not depend on write traffic.

Reply:
If the TiKV checks for all tablets' memtable too frequently, it will occupy too much CPU. If it checks not as frequently, the memory cannot be accurately controlled.

Reply:
> which is too big for fast recovery.

The recovery speed is not determined by the actual memtable size, but the amount of logs to read out. Because memtable write overhead is much smaller compared to read IO. That's why using age will reduce recovery time even though the total memtable size to recover might increase.